eng/pipelines/templates/stages/archetype-sdk-tests.yml

```diff
@@ -114,6 +114,8 @@ stages:
   - ${{ if or(contains(parameters.Clouds, cloud.key), and(contains(variables['Build.DefinitionName'], 'tests-weekly'), contains(parameters.SupportedClouds, cloud.key))) }}:
     - ${{ if not(contains(parameters.UnsupportedClouds, cloud.key)) }}:
       - stage: ${{ cloud.key }}_${{ parameters.TestName }}
+        variables:
+          SkipCoverage: ${{ not(eq(cloud.key, 'Public')) }}
```

Comment:
Cannot do this across the board. Maybe make the build and test steps customizable per cloud.